vowpalwabbit/config/include/vw/config/option.h

```diff
@@ -56,6 +56,25 @@ class base_option
   virtual void accept(typed_option_visitor& handler) = 0;
 
   virtual ~base_option() = default;
+
+  VW_ATTR(nodiscard) const std::vector<std::string>& get_tags() const { return _tags; }

```

Comment:
should we do lowercase only?

Reply:
Yeah that seems reasonable. Maybe c++ identifier rules as well